beetmoverscript/script.py

```diff
@@ -366,13 +369,17 @@ def list_bucket_objects(context, s3_resource, prefix):
 
     if update_balrog_manifest:
         context.raw_balrog_manifest.setdefault(locale, {})
+        balrog_info = generate_balrog_info(
+            context, artifact_pretty_name,
+            locale, destinations, from_buildid,
+        )
         if from_buildid:
-            component = 'partialInfo'
+            context.raw_balrog_manifest[locale].setdefault('partialInfo', []).append(balrog_info)
         else:
-            component = 'completeInfo'
-        context.raw_balrog_manifest[locale].setdefault(component, [])
-        context.raw_balrog_manifest[locale][component].append(generate_balrog_info(context, artifact_pretty_name,
-                                                                                   locale, destinations, from_buildid))
+            if update_balrog_manifest is True:
+                update_balrog_manifest = {'format': ''}
+            context.raw_balrog_manifest[locale].setdefault('completeInfo', {})[
+                update_balrog_manifest['format']] = balrog_info
```

Comment:
Judging by this and the change at line 355, I assume that, by default, in the absence of a format specified in the templates, the blob_suffix is not added. LGTM